inscriptions/ISic001402.xml

```diff
@@ -197,14 +197,15 @@
         <body>
             <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AL">
                 <ab>
-                    <lb n="1"/><persName type="attested"><name type="cognomen">Σατορ<choice><corr>ν</corr><sic>α</sic></choice><supplied reason="lost">ῖ</supplied>
+                    <lb n="1"/><persName type="attested"><name type="cognomen">Σατορν<supplied reason="lost">ῖ</supplied>
                     <lb n="2" break="no"/>λε</name></persName> χρησ<supplied reason="lost">τὲ</supplied>
                     <lb n="3"/><supplied reason="lost">χα</supplied>ῖρ<unclear>ε</unclear>
                 </ab>
             </div>
             <div type="apparatus">
                 <listApp>
                     <app><note>text according to IG</note></app>
+                    <app loc="line 1"><note>original transcription ΣΑΤΟΡΑ</note></app>
```

Comment:
properly this should be "anon. in Zeitschrift für die Alterthumswissenschaft 1844: ΣΑΤΟΡΑ"